includes/constants/class-country-codes.php

```diff
@@ -0,0 +1,230 @@
+<?php
+/**
+ * Class Country_Codes
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Constants;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Country Codes constants
+ *
+ * @psalm-immutable
+ */
+class Country_Codes extends Base_Constant {
```

Comment:
Can we use `Country` instead of `Country_Codes`?
It would be easier to read e.g. `Country::JAPAN`

Reply:
I've considered your point, but there's a potential for confusion. Some might expect `Country::JAPAN` to return 'Japan' or its equivalent in the currently configured language. To avoid this confusion, I opted for `Country_Code`.

Reply:
Makes sense, so I guess we can use `Country_Code`?

Reply:
Good point. Refactored in https://github.com/Automattic/woocommerce-payments/pull/7925/commits/d3ce810bdb73032b918a81dd53cb501f245ff213